src/components/ExpandedNode/ExpandedNode.tsx

```diff
@@ -162,10 +162,18 @@ const StatusField: FC<StatusFieldProps> = ({ documentation, isAccepted }) => {
   const rawDate = documentation.resource?.meta?.lastUpdated;
   if (rawDate) {
     const date = new Date(rawDate).toLocaleString('en-us');
+    let declinedText = '';
+    if (isAccepted) {
+      declinedText = status.charAt(0).toUpperCase() + status.slice(1);
+    } else if (isAccepted === null) {
+      declinedText = 'Status';
+    } else {
+      declinedText = 'Declined';
+    }
     return (
       <ExpandedNodeField
         key="Status"
-        title={isAccepted ? status.charAt(0).toUpperCase() + status.slice(1) : 'Declined'}
+        title={isAccepted ? status.charAt(0).toUpperCase() + status.slice(1) : declinedText}
```

Comment:
This line can probably just be `title={declinedText}` since you are already checking the `isAccepted` variable when you instantiate the value.  This can also probably be renamed to something like `titleText` since the text is not only for declined nodes.